modulemd/tests/test-modulemd-modulestream.c

```diff
@@ -2311,6 +2411,20 @@ main (int argc, char *argv[])
               module_stream_v2_test_rpm_map,
               NULL);
 
+  g_test_add ("/modulemd/v2/modulestream/v1/community",
```

Comment:
I'd suggest you to use `g_test_add_func()` like its done below because we have to change it into that anyway.